### PR TITLE
This ain't h2

### PR DESCRIPTION
**Pull request by martinthomson**
Try to make that a little clearer.  It seems like the motivation for asking for h2 examples was driven by the impression that this was somehow related to h2.  It's really just inspired by and I hope that this makes it clearer up front.

For #1924.

### CODE REVIEW

File: draft-ietf-httpbis-binary-message.md [hunk 1]
```diff
@@ -55,10 +55,9 @@ messages that can be conveyed outside of an HTTP protocol. This enables the
 transformation of entire messages, including the application of authenticated
 encryption.
 
-This format is informed by the framing structure of HTTP/2 ({{H2}}) and HTTP/3
-({{H3}}). In comparison, this format is simpler by virtue of not including
-either header compression ({{?HPACK=RFC7541}}, {{?QPACK=I-D.ietf-quic-qpack}})
-or a generic framing layer.
+The design of this format is a informed by the framing structure of HTTP/2
```
Comment (LPardue) [inline draft-ietf-httpbis-binary-message.md:58]:
```suggestion
The design of this format is informed by the framing structure of HTTP/2
```

**Review by LPardue — COMMENTED** [review]
I think you could go a bit futher here and say something along the lines that this isn't H2 but it is heavily reliant on certain requirements of H2, particularly about how things get serialized or parsed.

The problem you seem to be creating with this draft is like retreading HTTP/1.1 messaging defining HTTP semantic rules. There's things like pseudo-headers that now pop up in a few places and we're trying not to dupe the rules. Pointing to H2 all the time is mildly confusing and distracting but it might be the only thing we can do for now.

**Comment by martinthomson** [comment]
Good feedback.  Does that get you a little closer to the mark?

File: draft-ietf-httpbis-binary-message.md [hunk 2]
```diff
@@ -264,10 +262,9 @@ Request Control Data {
 
 ## Response Control Data
 
-The control data for a request message includes a single field that corresponds
-to the `:status` pseudo-header field in HTTP/2; see {{Section 8.3.2 of
-H2}}. This field is encoded as a single variable length integer, not a decimal
-string.
+The control data for a request message includes a single field that includes the
+status code.  This field is encoded as a single variable length integer, not a
+decimal string.
```
Comment (LPardue) [inline draft-ietf-httpbis-binary-message.md:267]: For consistency with the request para (and fixing incorrect use of `request message` here)

```suggestion
Response Control Data contains the status code of a response message. It is
encoded as a single variable length integer, not a decimal string.
```

File: draft-ietf-httpbis-binary-message.md [hunk 3]
```diff
@@ -235,15 +234,14 @@ Other values cause the message to be invalid; see {{invalid}}.
 
 ## Request Control Data
 
-The control data for a request message includes four values that correspond to
-the values of the `:method`, `:scheme`, `:authority`, and `:path` pseudo-header
-fields described in HTTP/2 ({{Section 8.3.1 of H2}}). These fields are
-encoded, each with a length prefix, in the order listed.
+Request Control Data contains the method and target of a request message. That
```
Comment (LPardue) [inline draft-ietf-httpbis-binary-message.md:237]: I notice now the capitalization here is incosistent with the rest of the body. I'll let you pick how to resolve that.

File: draft-ietf-httpbis-binary-message.md [hunk 4]
```diff
@@ -571,7 +565,14 @@ versions.  In particular, this format does not allow for:
 * framing of responses that depends on the corresponding request (such as HEAD)
   or the value of the status code (such as 204 or 304)
 
-Many of these same restrictions are shared by HTTP/2 {{H2}} and HTTP/3 {{H3}}.
+This format shares some of these properties with HTTP/2 and HTTP/3.
```
Comment (LPardue) [inline draft-ietf-httpbis-binary-message.md:568]: This sentence now reads a bit strangely in combination with the list above. In simple reading it goes "this format lacks X, Y, Z that are defined for H2 and H3 ... this format shares some of the features we just said are lacking". 

Perhaps rephrase this as "some of these features are similarly not supported by HTTP/2 and HTTP/3, making the features unique to HTTP/1.1"?

Comment (LPardue) [inline draft-ietf-httpbis-binary-message.md:568]: This sentence now reads a bit strangely in combination with the list above. In simple reading it goes "this format lacks X, Y, Z that are defined for H2 and H3 ... this format shares some of the features we just said are lacking". 

Perhaps rephrase this as "some of these features are similarly not supported by HTTP/2 and HTTP/3, making the features unique to HTTP/1.1"?

Comment (LPardue) [inline draft-ietf-httpbis-binary-message.md:568]: This sentence now reads a bit strangely in combination with the list above. In simple reading it goes "this format lacks X, Y, Z that are defined for H2 and H3 ... this format shares some of the features we just said are lacking". 

Perhaps rephrase this as "some of these features are similarly not supported by HTTP/2 and HTTP/3, making the features unique to HTTP/1.1"?

**Comment by martinthomson** [comment]
Fix and merged manually.
